Squablr/ExampleViewController.m

```diff
@@ -0,0 +1,440 @@
+/*
```

Comment:
What is this file doing / where did this code come from?

Reply:
I was using this as a resource on the google object I asked you.